platforms/shared/configuration/roles/create/shared_helm_component/templates/storage_class.j2

```diff
@@ -0,0 +1,20 @@
+metadata:
```

Comment:
why is this suddenly a .j2 file. Please convert to tpl

Reply:
I think we should move back to .tpl for templates, though from template engine perspective it doesn’t care about extension. Ansible seems go with .j2 but .tpl makes more sense to me as it says something about its use